integration/inspektor-gadget/integration_test.go

```diff
@@ -764,10 +765,27 @@ func TestOomkill(t *testing.T) {
 	t.Parallel()
 
 	oomkillCmd := &Command{
-		Name:           "StarOomkilGadget",
-		Cmd:            fmt.Sprintf("$KUBECTL_GADGET trace oomkill -n %s", ns),
-		ExpectedRegexp: `\d+\s+tail`,
-		StartAndStop:   true,
+		Name:         "StartOomkilGadget",
+		Cmd:          fmt.Sprintf("$KUBECTL_GADGET trace oomkill -n %s -o json", ns),
+		StartAndStop: true,
+		ExpectedOutputFn: func(output string) error {
+			expectedEntry := &oomkillTypes.Event{
+				Event:         BuildBaseEvent(ns),
+				KilledComm:    "tail",
+				TriggeredComm: "tail",
+			}
+			expectedEntry.Container = "test-pod-container"
+
+			normalize := func(e *oomkillTypes.Event) {
+				e.Node = ""
+				e.KilledPid = 0
+				e.Pages = 0
+				e.TriggeredPid = 0
+				e.MountNsID = 0
+			}
+
+			return ExpectEntriesToMatch(output, normalize, expectedEntry)
```

Comment:
I don't expect other OOM to happen, so what about using `ExpectAllToMatch`?

Reply:
Would be possible if `TriggeredComm` is always `tail` (The question)